wp-content/themes/csisnuclearnetwork/assets/_scss/base/_base.scss

```diff
@@ -26,7 +28,7 @@
 
 html,
 body {
-  // overflow-x: hidden;
+  overflow-x: hidden;
```

Comment:
Why was this added? Typically we try to avoid adding an `overflow: hidden` to the body because it breaks things like `position: sticky` which are used for other elements on the page.

Reply:
Grid-template-rows: MAX-CONTENT (edit)